### ISSUE TITLE
adjust_learning_rate seems buggy

### ISSUE THREAD

**Issue by cormoran** [mangye16/Unsupervised_Embedding_Learning#7]
Current implementation of `adjust_learning_rate`  in `demo_seen.py` decays lr only at 120 epoch (`elif ...` part seems unreachable).
Is it expected behavior ?

```
def adjust_learning_rate(optimizer, epoch):
    """Sets the learning rate to the initial LR decayed by 10 every 30 epochs"""
    lr = args.lr
    if epoch >= 120:
        lr = args.lr * 0.1
    elif epoch >= 160:
        lr = args.lr * 0.05
    elif epoch >= 200:
        lr = args.lr * 0.01
```

https://github.com/mangye16/Unsupervised_Embedding_Learning/blob/master/demo_seen.py#L160-L168

I also found the lr scheduing storategy is different from the paper in arXiv. Did you change it on purpose? or accidentally?

> The initial learning rate is set to 0.03, and it is decayed by 0.1 and 0.01 at 120 and 160 epoch

**Comment by mangye16** [comment]
Thanks for pointing out this bug. We will fix it. I think it would result in slightly different performance.
